### ISSUE TITLE
conda create: `Error: Failed to write to u'/opt/anaconda/2_7/pkgs/python-2.7.12-1.tar.bz2.part'`

### ISSUE THREAD

**Issue by connercowling** [conda/conda#3031]
I get this when trying to create a new conda environment for a project:

```
conda create --name <my-new-env> python=2.7
Fetching package metadata: ....
Solving package specifications: ..................
Package plan for installation in environment /opt/anaconda/2_7/envs/<my-new-env>:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    python-2.7.12              |                1        12.1 MB
    setuptools-23.0.0          |           py27_0         455 KB
    pip-8.1.2                  |           py27_0         1.5 MB
    ------------------------------------------------------------
                                           Total:        14.0 MB

The following NEW packages will be INSTALLED:

    openssl:    1.0.2h-1     
    pip:        8.1.2-py27_0 
    python:     2.7.12-1     
    readline:   6.2-2        
    setuptools: 23.0.0-py27_0
    sqlite:     3.13.0-0     
    tk:         8.5.18-0     
    wheel:      0.29.0-py27_0
    zlib:       1.2.8-3      

Proceed ([y]/n)? y

Fetching packages ...
Error: Failed to write to u'/opt/anaconda/2_7/pkgs/python-2.7.12-1.tar.bz2.part'. 
```

Here's the conda info output:

```
conda info
Current conda install:

             platform : linux-64
        conda version : 3.18.1
  conda-build version : 1.14.1
       python version : 2.7.10.final.0
     requests version : 2.7.0
     root environment : /opt/anaconda/2_7  (writable)
  default environment : /opt/anaconda/2_7
     envs directories : /opt/anaconda/2_7/envs
        package cache : /opt/anaconda/2_7/pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/linux-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/linux-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : None
    is foreign system : False

(root)ddsssuser@DDSSS3:~$ conda env list
# conda environments:
#
py2                      /opt/anaconda/2_7/envs/py2
py3                      /opt/anaconda/2_7/envs/py3
py3_4                    /opt/anaconda/2_7/envs/py3_4
smartvent                /opt/anaconda/2_7/envs/smartvent
root                  *  /opt/anaconda/2_7
```

**Comment by HugoTian** [comment]
I think this is a permission error, and will be fixed for conda 4.3.

you can try with sudo, see whether it works.

In future conda version, it will perform  check first, and if there is potential permission error, conda will exit with proper message.

**Comment by kalefranz** [comment]
Permissions errors and multi-user/read-only installs have been fixed in later versions of conda 4.3.  Also fixed with transactions in 4.3 is conda removing itself during a failed install/update operation.
